lib/shoryuken/fetcher.rb

```diff
@@ -32,7 +32,7 @@ def fetch(queue, available_processors)
           limit = batch ? FETCH_LIMIT : available_processors
 
           if (sqs_msgs = Array(receive_messages(queue, limit))).any?
-            logger.info { "Found #{sqs_msgs.size} messages for '#{queue}'" }
+            logger.debug { "Found #{sqs_msgs.size} messages for '#{queue}'" }
```

Comment:
@phstc  @mariokostelac  Is it really useful in production environment ? changing it to debug will not affect development environment, in the other hand, it will help a lot of people who are using paying solutions to manage logs ;-)
Not to mention the fact that we can analyze all the trafic for the last 2 weeks in the Amazon monitoring tab. And if we really want this kind of repetitive logs, it is very easy to add it in the worker.